File: mobile/lib/models/library.dart
class Library {
  final int id;
  final String name;
  final String? description;
  final int? dailyNewCards;
  final int? dailyReviewLimit;
  final String createdAt;
  final String updatedAt;

  Library({
    required this.id,
    required this.name,
    this.description,
    this.dailyNewCards,
    this.dailyReviewLimit,
    required this.createdAt,
    required this.updatedAt,
  });

  factory Library.fromJson(Map<String, dynamic> json) {
    return Library(
      id: json['id'],
      name: json['name'],
      description: json['description'],
      dailyNewCards: json['daily_new_cards'],
      dailyReviewLimit: json['daily_review_limit'],
      createdAt: json['created_at'],
      updatedAt: json['updated_at'],
    );
  }
}

File: backend/app/schemas/library.py
from pydantic import BaseModel, field_serializer
from datetime import datetime, timezone, timedelta
from typing import List, Optional

CN_TZ = timezone(timedelta(hours=8))

class LibraryCreate(BaseModel):
    name: str
    description: Optional[str] = None
    daily_new_cards: Optional[int] = None
    daily_review_limit: Optional[int] = None

class LibraryUpdate(BaseModel):
    name: Optional[str] = None
    description: Optional[str] = None
    daily_new_cards: Optional[int] = None
    daily_review_limit: Optional[int] = None

class LibraryResponse(BaseModel):
    id: int
    name: str
    description: Optional[str]
    user_id: int
    daily_new_cards: Optional[int]
    daily_review_limit: Optional[int]
    created_at: datetime
    updated_at: datetime

    @field_serializer('created_at', 'updated_at')
    def serialize_dt(self, v: datetime) -> str:
        return v.astimezone(CN_TZ).isoformat()

    class Config:
        from_attributes = True

class LibraryListResponse(BaseModel):
    total: int
    items: List[LibraryResponse]
